plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -515,6 +515,18 @@ struct Database {
         return app.data;
     }
 
+    /** Get SourceLoc for a specific mutation id.
+     */
+    SourceLoc getSourceLocation(MutationId id) @trusted {
+        auto stmt = db.prepare(format!"SELECT line, column FROM %s WHERE id IN (SELECT mp_id FROM %s WHERE id=:mut_id)"(
```

Comment:
same here, I recommend moving the SQL to a line before this and assigned to an immutable variable. forces it to be computed at compile time and only once.